packages/block-editor/src/components/button-block-appender/index.js

```diff
@@ -42,7 +42,7 @@ function ButtonBlockAppender(
 					);
 				} else {
 					label = _x(
-						'Add block',
+						'Add new item',
```

Comment:
This label is still generic. From an accessibility standpoint, it would be nice to hear what I am adding to what. It seems like it would make more sense to have something like this.

label = sprintf( _x( 'Add block to %s' ), parent_block_title );

Not sure how hard this would be to implement, but seems worth the time if we're already here.